src/lib/email-service.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable class-methods-use-this */
+const nodemailer = require("nodemailer")
+
+module.exports = class Email {
```

Comment:
cool

curious as to the reason for the class approach here. perhaps as opposed having `createTransport` at the top and using it to send mail

Reply:
@jocrah with this class, we can define as many other new mail sending methods as we want and they can be called directly from the class.

Reply:
@itzgeebee  okay its a bit clearer but need a little more clarification. can you give an example of some other mail-sending methods in this case? because we're using basically same envs in the `newTransport` so not seeing how it allows for other ones

Reply:
@itzgeebee okay got it

I don't think we need to need to a new transport` for each sending when it basically the same though or there's something to it?

Reply:
@jocrah I'll change it up then

Reply:
@jocrah I get. I'll do it this way.

Reply:
@jocrah I have updated it.

Reply:
great taking a look